project/Dependencies.scala

```diff
@@ -6,79 +6,80 @@ object Dependencies {
 
   object Version {
 
-    val parCollections = "1.0.0"
+    val parCollections = "1.0.3"
 
-    val akka = "2.6.12"
-    val akkaHttp = "10.2.3"
+    val akka = "2.6.15"
+    val akkaHttp = "10.2.6"
 
-    val scalaTest = "3.2.3"
-    val scalaCheck = "1.15.2"
-    val scalaTestPlusCheck = "3.2.3.0"
+    val scalaTest = "3.2.9"
+    val scalaCheck = "1.15.4"
+    val scalaTestPlusCheck = "3.2.9.0"
     val scalaMock = "5.1.0"
     val diffx = "0.3.29" // Can't update to 0.3.30, we have an issue with child traits of AcceptedOrder
 
-    val cats = "2.1.1"
-    val catsTaglessMacros = "0.12"
+    val cats = "2.6.1"
+    val catsMacros = "2.1.1"
+    val catsTaglessMacros = "0.14.0"
     val kindProjector = "0.9.10"
     val betterMonadicFor = "0.3.1"
-    val mouse = "0.26.2"
-    val shapeless = "2.3.3"
+    val mouse = "1.0.4"
+    val shapeless = "2.3.7"
     val monocle = "2.1.0"
 
     val typesafeConfig = "1.4.1"
-    val scopt = "4.0.0"
+    val scopt = "4.0.1"
 
-    val logback = "1.2.3"
-    val slf4j = "1.7.30"
-    val janino = "3.1.3"
+    val logback = "1.2.5"
+    val slf4j = "1.7.32"
+    val janino = "3.1.6"
     val logbackJsonEncoder = "6.6"
 
-    val silencer = "1.7.2"
+    val silencer = "1.7.5"
 
     val kanela = "1.0.11"
-    val kamon = "2.2.0"
+    val kamon = "2.2.3"
 
     val wavesProtobufSchemas = "1.3.1"
-    val wavesJ = "1.0.1"
+    val wavesJ = "1.2.2"
 
-    val postgresql = "42.2.18"
-    val quillJdbc = "3.6.0"
+    val postgresql = "42.2.23"
+    val quillJdbc = "3.9.0"
 
-    val sttpClient = "3.2.3"
+    val sttpClient = "3.3.13"
 
-    val testContainers = "0.39.3"
-    val testContainersPostgres = "1.15.2"
-    val testContainersKafka = "1.15.2"
-    val testContainersToxiProxy = "1.15.2"
+    val testContainers = "0.39.5"
+    val testContainersPostgres = "1.16.0"
+    val testContainersKafka = "1.16.0"
+    val testContainersToxiProxy = "1.16.0"
 
     val jackson = "2.10.0"
     val playJson = "2.9.2"
 
     val googleGuava = "28.2-jre"
-    val kafka = "2.7.0"
+    val kafka = "2.8.0"
 
-    val swagger = "1.1.2"
-    val swaggerUi = "3.32.5"
+    val swagger = "1.4.0"
+    val swaggerUi = "3.51.2"
     val jaxbApi = "2.3.1"
 
     val scorexCrypto = "2.1.10"
 
-    val monix = "3.3.0"
+    val monix = "3.4.0"
 
     val supertagged = "1.5"
 
     val javaLevelDb = "0.12"
     val iq80levelDb = "0.12"
     val influxDb = "2.21"
-    val levelDbVersion = "1.22.1"
+    val levelDbVersion = "1.23.0"
 
-    val commonsNet = "3.7.2"
+    val commonsNet = "3.8.0"
     val nettyCodec = "4.1.33.Final"
     val jwt = "5.0.0"
 
-    val pureConfig = "0.14.0"
-    val allureScalaTest = "2.13.8"
-    val enumeratum = "1.6.1"
+    val pureConfig = "0.16.0"
+    val allureScalaTest = "2.13.9"
+    val enumeratum = "1.7.0"
 
     val scalaPbJson = "0.9.3"
```

Comment:
Updating causes `waves-integration-it` tests failures (class scalapb/Message not found - wrong versions of scalapb may cause it, so decided to leave it as it is)

Reply:
Let's write a comment about this.